spec/features/users/profile_recovery_for_gpo_verified_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.feature 'Password recovery via personal key for a GPO-verified user' do
+  include IdvStepHelper
+
+  let(:user) { create(:user, :fully_registered) }
+  let(:new_password) { 'some really awesome new password' }
+
+  before do
+    allow(FeatureManagement).to receive(:reveal_gpo_code?).and_return(true)
+  end
+
+  scenario 'lets them reactivate their profile with their personal key', email: true, js: true do
+    complete_idv_steps_with_gpo_before_confirmation_step(user)
+    click_on 'Continue'
+
+    click_on t('account.index.verification.reactivate_button')
+    click_on t('idv.gpo.form.submit')
```

Comment:
At this point, you need to:
- capture the GPO code
- sign out
- sign back in
- enter the gpo code
- scrape personal key and continue

Reply:
(The key is signing out after entering GPO but _before_ entering the code.

Reply:
The goal should be for this test to _fail_ after when run against d67eaa2eccd9967af953598f70100dfbf3bde4b2